bite-size-notes/week-9-d-CKF.md

```diff
@@ -0,0 +1,136 @@
+# Introduction
+
+## Scope + Objectives
+
+These notes aim to provide a concise yet thorough introduction to
+continuous Kalman Filters (CKFs). Readers will learn the fundamentals of
+CKFs, their applications in control systems, and how they differ from
+discrete Kalman Filters. Key questions addressed include:
+-   What is a continuous Kalman Filter?
+-   How is it formulated mathematically?
+
+
+## Introduction
+
+Continuous Kalman Filters are essential in control theory and signal
+processing for estimating the state of a continuous-time dynamic system.
+They are particularly useful in applications requiring real-time state
+estimation, such as navigation systems, robotics, and autonomous
+vehicles. The continuous Kalman Filter extends the discrete Kalman
+Filter to continuous-time processes, offering more accurate and smooth
+state estimates for systems modeled by differential equations.
+
+## Continuous-Time Kalman Filter
+
+We start with the continuous time dynamical system with Gaussian noise:
+$$\dot{x}(t) = Ax(t) + Bu(t) + w(t)  \quad\quad\quad\quad w(t) \sim \mathcal{N}(0,Q)$$ 
+$$y(t) = Cx(t) + Du(t) + v(t) \quad\quad\quad\quad v(t) \sim \mathcal{N}(0,R)$$
+where, $Q$ is Process noise covariance and *R* is measurement noise
+covariance. We also assume that our measurement is not affected by the
+control, so $D = 0$.
+
+To derive the Kalman filter equations for these dynamics, we employ
+discrete-time dynamics and approximate the equations accordingly. We
+write the derivatives as finite difference terms as shown below,
+$$\\dot{x} \\approx \\frac{x\_{t+1}-x\_{t}}{\\Delta t}$$
+where $Δt → 0$.
+
+Substituting this into continuous dynamics, we get,
+
+$$\\frac{x\_{t+1}-x_t}{\\Delta t} = Ax\_{t} + Bu\_{t} + w$$
+$$x\_{t+1} = (I+A \\Delta t)x\_{t} + B \\Delta t u\_{t} + \tilde{w}  \\quad \\quad \\tilde{w} \\sim N(0,Q\\Delta t)$$
+Also,
+$$y\_{t} = Cx\_{t} + \\tilde{v} \\quad \\tilde{v} \\sim N(0,\\frac{R}{\\Delta t})$$
+
+The noise $w(t)$ is integrated over $Δt$ The resulting covariance of
```

Comment:
Why do the noise terms get treated differently than the other terms?  The noise terms get integrated from $t$ to $t+ \Delta t$ while the other terms simply get multiplied by $\Delta t$.  Why?

Reply:
So, all other terms are also integrated assuming zero order hold, control is constant over dt time interval. But we cant say that for a random noise.